znoyder/generator.py

```diff
@@ -180,10 +185,15 @@ def main(argv=None) -> None:
         else:
             ds_name = us_to_ds_projects_map[name]
 
-        config_dest = os.path.join(
-            GENERATED_CONFIGS_DIR,
-            CONFIG_PREFIX + ds_name + CONFIG_EXTENSION
-        )
+        if not args.aggregate:
+            config_dest = os.path.join(
+                GENERATED_CONFIGS_DIR,
+                CONFIG_PREFIX + ds_name + CONFIG_EXTENSION
+            )
+            write_mode = 'w'
+        else:
+            config_dest = os.path.join(GENERATED_CONFIGS_DIR, args.aggregate)
+            write_mode = 'a'
```

Comment:
As I understand, the goal in this case is to have all output in the single file, rather than in multiple ones?
I am concerned here about idempotency – right now running generation with --aggregate option multiple times will duplicate the file content. Shouldn't the file be cleaned at some point before going here?

Reply:
Added code to cleanup the generated-jobs directory every run

Reply:
:+1: Works for me